tools/autograd/gen_variable_type.py

```diff
@@ -350,6 +350,14 @@
 return c10::Dispatcher::singleton().redispatch<${ret_and_arg_types}>(${profiled_dispatch_args});
 """)
 
+
+# TraceType templates
+TRACE_DISPATCH_UNBOXED = CodeTemplate("""\
+static auto op = c10::Dispatcher::singleton().findSchemaOrThrow("aten::${operator_name}", "${overload_name}");
```

Comment:
No action needed for this PR, but we really need to come up with a better idiom for redispatching in these situations.